packages/components/organisms/f-checkout/src/components/_tests/Accordion.test.js

```diff
@@ -0,0 +1,87 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Accordion from '../Accordion.vue';
+import { i18n, createStore } from './helpers/setup';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('Accordion', () => {
+    const propsData = { title: 'Accordion Header', id: 'note' };
+
+    it('should be defined', () => {
+        // Arrange
+        const wrapper = shallowMount(Accordion, {
+            i18n,
+            store: createStore(),
+            localVue,
+            propsData
+        });
+
+        // Assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    describe('methods ::', () => {
+        describe('toggle ::', () => {
+            let wrapper;
+
+            it('should change `isExpanded` to true when it is initally set to false', () => {
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: false
+                        };
+                    }
+                });
+
+                wrapper.vm.toggle();
+                expect(wrapper.vm.$data.isExpanded).toBe(true);
+            });
+
+            it('should change `isExpanded` to false when it is initally set to true', () => {
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: true
+                        };
+                    }
+                });
+
+                wrapper.vm.toggle();
+                expect(wrapper.vm.$data.isExpanded).toBe(false);
+            });
+
+            it('should be called when the accordion header is clicked', () => {
+                const toggleSpy = jest.spyOn(Accordion.methods, 'toggle');
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: false
+                        };
+                    }
+                });
+
+                const header = wrapper.find('[data-test-id="note-accordion-header"]');
+                header.trigger('click');
+                expect(toggleSpy).toHaveBeenCalled();
+                expect(wrapper.vm.$data.isExpanded).toBe(true);
+            });
```

Comment:
This is not really a unit test for `toggle`, in my humble opinion. I'd place this test outside `methods`, under a `template ::` describe, similar to what we've done in other components within Checkout.

Reply:
Done